changelog.md

```diff
@@ -23,7 +23,7 @@ Bug Fixes:
 
 1.27.1 (2024/03/28)
 ===================
-
+* Added fzf history search functionality. The feature can switch between the old implementation and the new one based on the presence of the fzf binary.
```

Comment:
I'm sorry, the release notes for the implemented feature should be written in the upcoming release section, not in the already released 1.27.1. I will correct this along with the CI.

Reply:
Fixed in [e766643](https://github.com/dbcli/mycli/pull/1170/commits/e766643982a6062043767904afdf6e893050a0cb)